### DISCUSSION TITLE
Responsive images are not generated with spatie-laravel-media-library-plugin

### DISCUSSION THREAD

**Discussion by yabasha** [filamentphp/filament#12255] [Help]
### Package

Panel builder

### Package Version

v3.2.63

### How can we help you?

Responsive images are not generated, I am following the instructions in: [the plugin page](https://filamentphp.com/plugins/filament-spatie-media-library#generating-responsive-images)

```
SpatieMediaLibraryFileUpload::make('attachments')
    ->multiple()
    ->responsiveImages()
```

Whatever I do the responsive array is always empty:

<img width="784" alt="image" src="https://github.com/filamentphp/filament/assets/932321/4215555e-99bc-4a21-8b9b-e529332a70e1">

Please Advice,

**Comment by madushancs** [comment]
Hi [yabasha](https://github.com/yabasha). Have run queue worker?

**Reply by eskayamadeus** [reply]
This is the right solution. Should've been stated in the docs.
